src/zepben/eas/client/work_package.py

```diff
@@ -358,6 +432,16 @@ class ModelConfig:
     The name of the set of distribution transformer tap settings to be applied to the model from an external source.
     """
 
```

Comment:
If you look at the class this is not the correct spot to put the override. Remember that the data objects for the EAS Client and the data objects we use internally are not the same. We do this to make the interface nicer for the customer.

If you look at the task the load overrides go into the FixedTime and TimePeriod classes.

Reply:
moved it as per spec